packages/cli/src/commands/create-plugin/createPlugin.ts

```diff
@@ -174,8 +180,16 @@ export async function movePlugin(
   });
 }
 
-export default async () => {
+export default async (cmd: Command) => {
   const codeownersPath = await getCodeownersFilePath(paths.targetRoot);
+  let npmScope = '';
+  if (cmd.scope) {
+    if (!/^@[a-z0-9]+(-[a-z0-9]+)*$/.test(cmd.scope)) {
```

Comment:
I think we can just do a simple `.startsWith('@')` here?

Reply:
fixed. thanks.